workflows/methylseq.nf

```diff
@@ -204,6 +232,25 @@ workflow METHYLSEQ {
         ch_bam
     )
     versions = versions.mix(PRESEQ_LCEXTRAP.out.versions.first())
+    
+    /*
+     * MODULE: HS METRICS.
+     */ 
+    if (targetIntervalProvided){
```

Comment:
```suggestion
    if (params.target_interval){
```
you can provide the param here directly without having to declare another variable

Reply:
or could do something like [this](https://github.com/nf-core/methylseq/pull/303/files#diff-e2ab58f278a961067539c377b7bae820d21cc18befde5f2d40ce6576bbbd9debR196)